tests/server/schemas/test_schedules.py

```diff
@@ -25,7 +25,7 @@
 
 class TestCreateIntervalSchedule:
     def test_interval_is_required(self):
-        with pytest.raises(ValidationError):
+        with pytest.raises(ValidationError, match="(Field required)"):
```

Comment:
I think we should be explicit here since the field being required is what the test is testing